### PR TITLE
DSGOV-322-Simplify-Paths

### PR DESCRIPTION
**Pull request by afcalderonm**
## Description
simplify route paths

[Audit Service: API Clean Up - Simplify Paths](https://dol-ewf.atlassian.net/browse/DSGOV-322)

## Motivation and Context

change the paths to be a more RestFull 

## How Has This Been Tested?
tested with postman consuming all the changes paths

## Types of changes

Patch

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [x] Refactor or cleanup (functionality unchanged)
  Minor
- [ ] New feature (non-breaking change which adds functionality)
  Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [x] I have confirmed all acceptance criteria defined by the story have been met.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -33,7 +34,7 @@ paths:
     get:
       summary: Gets all audit events for the specified business object
       operationId: getEvents
-      tags: ['Audit Events']
+      tags: ['audit-events']
```
Comment (asultan001) [inline swagger.yaml:37]: I don't think we should change the tag name here

File: swagger.yaml [hunk 2]
```diff
@@ -81,7 +82,7 @@ paths:
     post:
       summary: Creates an audit event for the specified business object
       operationId: postEvent
-      tags: ['Audit Events']
+      tags: ['audit-events']
```
Comment (asultan001) [inline swagger.yaml:85]: I don't think we should change the tag name here

**Review by asultan001 — APPROVED** [review]
@afcalderonm LGTM just left a small comment.

File: swagger.yaml (hunk 1 above)
Comment (JayStGelais) [reply swagger.yaml:37]: @asultan001 I think the change here is what eliminated the repeating entries in the API docs.

**Review by JayStGelais — APPROVED** [review]

**Comment by afcalderonm** [comment]
@JayStGelais can you help me out approving the changes again? it's need because i have to update the branch to merge it and it's asking me again for an approval

File: swagger.yaml (hunk 1 above)
Comment (asultan001) [reply swagger.yaml:37]: Yeah in that case I suppose I'm cool with it!

**Review by asultan001 — APPROVED** [review]
LGTM

**Pull request merged**